src/shared/lib/isSampleProfiled.ts

```diff
@@ -66,13 +66,18 @@ export function isSampleProfiledInMultiple(
     coverageInformation: CoverageInformation,
     hugoGeneSymbol?: string
 ): boolean[] {
-    // returns boolean[] in same order as molecularProfileIds
-    const profiledReport = getSampleProfiledReport(
-        uniqueSampleKey,
-        coverageInformation,
-        hugoGeneSymbol
-    );
-    return molecularProfileIds.map(
-        molecularProfileId => !!profiledReport[molecularProfileId]
-    );
+    // returns empty list if molecularProfileIds is undefined
+    if (!molecularProfileIds) {
```

Comment:
since this logic is in here then the molecularProfileIds param should be made possibly undefined for consistency

Reply:
Thanks! updated.